### PR TITLE
Feature/dynamic route

### PR DESCRIPTION
**Pull request by Eventyret**
Changed something

### CODE REVIEW

**Comment by Eventyret** [comment]
Commetns won't do much :)

File: app.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
 @APP.route('/')
 def hello_world():
     return render_template('pages/index.html', title="The Lazy Padwan and his lost Son")
```
Comment (Eventyret) [inline app.py:15]: Was this the title we wanted to change ?

File: app.py [hunk 2]
```diff
@@ -13,6 +13,10 @@
...
+@APP.route('/<name>')
+def hello_user(name):
+    return render_template('pages/name.html', title="The Lazy Padwan", submittedName=name)
```
Comment (Eventyret) [inline app.py:18]: This the title you wanted changed ?

Comment (OlekSt) [reply app.py:18]: Yes, let's make it more descriptive.

Comment (Eventyret) [reply app.py]: This has now been updated if completed please resolve conversation

**Pull request merged**